pkg/controller/prpqr_reconciler/prpqr_reconciler.go

```diff
@@ -299,7 +301,7 @@ func (r *reconciler) triggerJobs(ctx context.Context,
 			prowjobsToCreate = append(prowjobsToCreate, aggregatedProwjobs...)
 
 			submitted := generateJobNameToSubmit(inject, pullRequests)
-			aggregatorJob, err := generateAggregatorJob(baseMetadata, uid, mimickedJob, jobSpec.JobName(jobconfig.PeriodicPrefix), req.Name, req.Namespace, r.prowConfigGetter, time.Now(), submitted)
+			aggregatorJob, err := generateAggregatorJob(r, baseMetadata, uid, mimickedJob, jobSpec.JobName(jobconfig.PeriodicPrefix), req.Name, req.Namespace, r.prowConfigGetter, time.Now(), submitted)
```

Comment:
nit: `generateAggregatorJob` really shouldn't take the reconciler as a parameter. It would be better if it just took the duration directly instead. (like we are doing with the `startTime`, for instance